modules/renter/download_test.go

```diff
@@ -20,6 +23,8 @@ type testHost struct {
 	// used to simulate real-world conditions
 	delay    time.Duration // transfers will take this long
 	failRate int           // transfers will randomly fail with probability 1/failRate
+
+	sync.Mutex
```

Comment:
not that there aren't equally complex testing constructions in other places, wanted to call it out anyway